packages/eas-cli/src/project/publish.ts

```diff
@@ -118,6 +118,11 @@ export async function convertAssetToUpdateInfoGroupFormatAsync(
 ): Promise<PartialManifestAsset> {
   const fileSHA256 = getBase64URLEncoding(await calculateFileHashAsync(asset.path, 'sha256'));
   const contentType = asset.contentType;
+
+  let fileExtension;
```

Comment:
Adding validation for fileExtension server side as well: https://github.com/expo/universe/pull/7970